notebooks/data-sources/TestGrid/metrics/time_to_test.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Here as well. Do we want to also alter the parameter name to avoid confusion?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>updated!</p>